packages/now-go/test/fixtures/14-go-mod-sub/now.json

```
@@ -0,0 +1,5 @@
+{
+  "version": 2,
+  "builds": [{ "src": "api/**/*.go", "use": "@now/go" }],
+  "probes": [{ "path": "/api", "mustContain": "RANDOMNESS_PLACEHOLDER" }]
```

```suggestion
  "probes": [
    { "path": "/api", "mustContain": "api:RANDOMNESS_PLACEHOLDER" },
    { "path": "/other-folder", "mustContain": "other:RANDOMNESS_PLACEHOLDER" }
  ]
```

Let's test that these are deployed properly